Library/Formula/trafficserver.rb

```diff
@@ -10,21 +19,39 @@ class Trafficserver < Formula
     sha256 "4920a708ef78452027f9a258f50b7802b3f0bdc68aca92337d2468d9abe88415" => :mountain_lion
   end
 
+  option "with-spdy", "Build with SPDY protocol support"
+  option "with-curl", "Build with libcurl from Homebrew"
```

Comment:
Hmm, I initially added it to support Leopard and then later I noticed that Leopard (because libcurl 7.19+ is required and it ships with 7.16)  is no longer supported. Given that it only seems to be used by tstop, which should work just as well with any compatible system curl I'll drop the option.